### PR TITLE
Testing with Jest and Enzyme

### PR DESCRIPTION
**Pull request by ecostrong**

### CODE REVIEW

File: cat-tinder-frontend/src/components/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()}) 
+
+describe("When the Footer renders", ()=> {
+    it("shows 4 NavItems", ()=> { 
+        const renderedFooter= shallow(<Footer/>)
+
+        const renderedNavItems = renderedFooter.find("NavItem")
+
+        expect(renderedNavItems.length).toEqual(0)
```
Comment (elysemontano) [inline cat-tinder-frontend/src/components/Footer.test.js]: Can you modify this test so it is checking for something?  What do you have on your footer that you can check?

**Review by elysemontano — CHANGES_REQUESTED** [review]
Currently, you aren't really checking for anything in your Header, Footer, and NotFound page tests.  Can you add some kind of a tag that you can check against and adjust your tests accordingly?

File: cat-tinder-frontend/src/components/Footer.test.js (hunk 1 above)
Comment (ecostrong) [reply cat-tinder-frontend/src/components/Footer.test.js]: Made up dates, added Nav Items to the header and footer.

File: cat-tinder-frontend/src/components/Footer.js [hunk 2]
```diff
@@ -1,9 +1,42 @@
...
+}
+export default Footer
+
+
+
+
+
+
+
+// export default class Footer extends Component {
```
Comment (sjproctor) [inline cat-tinder-frontend/src/components/Footer.js]: Please remove commented out code.

File: cat-tinder-frontend/src/components/Header.js [hunk 3]
```diff
@@ -1,12 +1,43 @@
...
+          </NavItem>
+      </Nav>
+    )
+  }
+}
+export default Header
+
+
+
+// export default class Header extends Component {
```
Comment (sjproctor) [inline cat-tinder-frontend/src/components/Header.js]: Please remove all commented out code.

File: cat-tinder-frontend/src/components/Footer.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()}) 
+
+describe("When the Footer renders", ()=> {
+    it("shows 4 NavItems", ()=> { 
+        const renderedFooter= shallow(<Footer/>)
+
+        const renderedNavItems = renderedFooter.find("NavItem")
+
+        expect(renderedNavItems.length).toEqual(4)
```
Comment (elysemontano) [inline cat-tinder-frontend/src/components/Footer.test.js]: I might be mistaken, I only see 3 NavItems in your Footer.  Does this test pass?

File: cat-tinder-frontend/src/components/Header.js [hunk 5]
```diff
@@ -1,13 +1,43 @@
...
+    )
+  }
 }
+export default Header
+
+
 
-export default Header;
\ No newline at end of file
+// export default class Header extends Component {
```
Comment (elysemontano) [inline cat-tinder-frontend/src/components/Header.js]: I would recommend removing the commented out code.

File: cat-tinder-frontend/src/pages/NotFound.test.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()})
+
+describe("When the NotFound renders", ()=> {
+  it("shows 4 NavItems", ()=> {
+      const renderedNotFound = shallow(<NotFound/>)
+
+      const renderedNavItems = renderedNotFound.find("NavItem")
+
+      expect(renderedNavItems.length).toEqual(0)
```
Comment (elysemontano) [inline cat-tinder-frontend/src/pages/NotFound.test.js]: This looks like it's not really testing anything.  Please update this test.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left a few more changes on here.  Also, it looks like you may be adding node_modules into this.  Can you check your .gitignore to make sure that node_modules are on there?

**Review by sjproctor — COMMENTED** [review]
It looks like there is only one test here. Is that the goal here? 
Also make sure you are removing all commented out code.

**Review by elysemontano — APPROVED** [review]
You got this!

**Pull request merged**